bin/runex

```diff
@@ -22,4 +22,4 @@ if command -v accumulo > /dev/null 2>&1 ; then
   av_arg="-Daccumulo.version=`accumulo version`"
 fi
 
-mvn -q exec:java -Dexec.mainClass="org.apache.accumulo.examples.$main_class" $av_arg -Dexec.args="$main_args"
+mvn -q exec:java -Dexec.mainClass="org.apache.accumulo.examples.$main_class" $av_arg -Dexec.args="$main_args" -Dlog4j.configuration=file:"./src/main/resources/log4j.properties"
```

Comment:
I will run it again without the -D to make sure its on the classpath.

Yeah I realized it was a relative path but the runex command doesn't work outside the root of the project anyway...